test/Release/Gateway/ConfigurableVisibilityReleaseGatewayMock.php

```diff
@@ -0,0 +1,55 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: Volodymyr
+ * Date: 04/08/2015
+ * Time: 12:17
+ */
+
+namespace Clearbooks\Labs\Release\Gateway;
+
+
+use Clearbooks\Labs\Release\Release;
+
+class ConfigurableVisibilityReleaseGatewayMock implements ReleaseGateway
+{
+
+    /**
+     * @var
+     */
+    private $visibility;
+
+    function __construct( $visibility = true )
+    {
+
+        $this->visibility = $visibility;
+    }
+
+
+    /**
+     * @param $releaseName
+     * @param $url
+     * @return int
+     */
+    public function addRelease( $releaseName, $url )
+    {
+        // TODO: Implement addRelease() method.
```

Comment:
I think a lint should be set up if you want to enforce this. I am not sure if PHP CodeSniffer or something like this would do the job.